src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Simplification/SymbolAnnotation.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.CodeAnalysis.Simplification
     /// <summary>
     /// An annotation that holds onto information about a type or namespace symbol.
     /// </summary>
-    internal class SymbolAnnotation
+    internal static class SymbolAnnotation
```

Comment:
Is there an analyzer to flag this?

Reply:
@mavasani 
https://github.com/dotnet/roslyn-analyzers/blob/2928b857198370bf518b8e5647188d2a055f5928/src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/StaticHolderTypes.cs#L44-L52

Not sure why it's disabled by default though.

I opened https://github.com/dotnet/roslyn/pull/65830 to enable it in this repo.